AirCasting/MicrophoneSession/MicrophoneManager.swift

```diff
@@ -120,7 +120,8 @@ private extension MicrophoneManager {
                                        thresholdHigh: 80,
                                        thresholdMedium: 70,
                                        thresholdLow: 60,
-                                       thresholdVeryLow: 20)
+                                       thresholdVeryLow: -100)
+        #warning("Change thresholdVeryLow to 20")
```

Comment:
Could You add ToDo: ->  #warning("ToDo: Change thresholdVeryLow to 20")

Reply:
Sure!